requirements.txt

```
@@ -50,55 +53,55 @@ Mako==1.2.4
 Markdown==3.4.1
 MarkupSafe==2.0.1
 messytables==0.15.2
+mypy==1.3.0
+mypy-extensions==1.0.0
 newrelic==8.8.0
 nose==1.3.7
 openpyxl==3.1.2
-packaging==23.0
-passlib==1.7.3
-PasteDeploy==2.0.1
+passlib==1.7.4
 pathtools==0.1.2
-pika==1.3.1
-pip==23.0.1
-polib==1.0.7
-psycopg2==2.8.6
+pika==1.3.2
+pip==23.1.2
+polib==1.1.1
+psycopg2==2.9.3
 pycparser==2.21
 PyJWT==2.4.0
 pyOpenSSL==23.2.0
 pysaml2==7.0.1
-pysolr==3.6.0
+pysolr==3.9.0
 python-dateutil==2.8.2
-python-editor==1.0.4
-python-magic==0.4.15
-pytz==2016.7
-PyUtilib==5.7.1
-PyYAML==5.4
-redis==4.5.4
-repoze.lru==0.7
-repoze.who==2.3
-requests==2.31.0
+python-magic==0.4.27
+pytz==2023.3
+pytz-deprecation-shim==0.1.0.post0
+PyUtilib==6.0.0
+PyYAML==6.0
+redis==4.5.5
+requests==2.28.1
```

```suggestion
requests==2.31.0
```
Took the wrong version of `requests` (different file)